docs(grid, treelist):added section for searchbox in the state of the … (#613)
telerik/blazor-docs: components/treelist/filter/searchbox.md

![treelist search box](images/search-box-overview.gif)

## Filter From Code

> You can set the TreeList filters from your code through the component [state]({%slug treelist-state%}).

Reviewer: ```suggestion
You can set the TreeList filters programmatically through the component [state]({%slug treelist-state%}).
```
